packages/cli/tests/e2e/bot/CommandExecutionIsUserError.ts

```diff
@@ -53,7 +53,7 @@ describe("Error type should be expected", function () {
       await CliHelper.deployProject(ResourceToDeploy.Bot, projectPath);
     } catch (e) {
       chai.assert.isTrue(e instanceof PluginError);
-      chai.assert.isTrue(e.ErrorType == ErrorType.User);
+      chai.assert.isTrue(e.ErrorType == "User");
```

Comment:
Nit: Can you share some insights for this change (use plain string to replace enum)?

Reply:
typescript's Enum is not recommended by every TS book,  Their behavior is weird and strange.
see more in [this](https://blog.logrocket.com/why-typescript-enums-suck/) , more weird action: https://pbs.twimg.com/media/FMHHwd9VUAICzro?format=jpg&name=large
using literal types is more recommended in TS

Reply:
changed to other code type